tests/suite/test_transport_server_tcp_load_balance.py

```diff
@@ -407,3 +408,121 @@ def test_tcp_request_load_balanced_method(
             client.close()
 
         assert len(endpoints) is 3
+
+    @pytest.mark.skip_for_nginx_oss
+    def test_tcp_passing_healthcheck_with_match(
```

Comment:
@pleshakov these tests also looks like a decent candidates to use parameterization to pass various ts/cases, see: https://github.com/nginxinc/kubernetes-ingress/blob/master/tests/suite/test_ingress_mtls.py#L82

Reply:
makes sense. let me know about this https://github.com/nginxinc/kubernetes-ingress/pull/1633/files#r643518158

Reply:
Yeah that's fine for now that its added and working but we don't need to verify all the attribs as nginx plus is doing lb and the UI is provided by the same.